packages/agent-toolkit/src/monday-graphql/queries.graphql.ts

```diff
@@ -559,3 +559,169 @@ export const getWorkspaceInfo = gql`
     }
   }
 `;
+
+// -----------------------------
+// WorkForms (Forms) Operations
+// -----------------------------
+
+// Create a new monday form (API version 2025-10)
+export const createForm = gql`
```

Comment:
Consider extracting those queries and placing them in a dedicated module under the `workforms-tools` directory. For reference, follow the structure of the `get-board-info` tool https://github.com/mondaycom/mcp/tree/master/packages/agent-toolkit/src/core/tools/platform-api-tools/get-board-info